platform/mellanox/iproute2/Makefile

```diff
@@ -0,0 +1,49 @@
+SHELL = /bin/bash
+.ONESHELL:
+.SHELLFLAGS += -e
+
+MAIN_TARGET = iproute2-mlnx_$(IPROUTE2_MLNX_VERSION_FULL)_$(CONFIGURED_ARCH).deb
+IPROUTE2_DEB = iproute2_$(IPROUTE2_MLNX_VERSION_FULL)_$(CONFIGURED_ARCH).deb
+IPROUTE2_TMP := $(shell mktemp -u -d -t ipr.XXXXXXXXXX)
+
+$(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
+	# Remove any stale files
+	rm -rf iproute2-$(IPROUTE2_MLNX_VERSION)
+
+	wget -O iproute2_$(IPROUTE2_MLNX_VERSION).orig.tar.xz http://deb.debian.org/debian/pool/main/i/iproute2/iproute2_$(IPROUTE2_MLNX_VERSION).orig.tar.xz
+	wget -O iproute2_$(IPROUTE2_MLNX_VERSION_FULL).dsc http://deb.debian.org/debian/pool/main/i/iproute2/iproute2_$(IPROUTE2_MLNX_VERSION_FULL).dsc
+	wget -O iproute2_$(IPROUTE2_MLNX_VERSION_FULL).debian.tar.xz http://deb.debian.org/debian/pool/main/i/iproute2/iproute2_$(IPROUTE2_MLNX_VERSION_FULL).debian.tar.xz
+	dpkg-source -x iproute2_$(IPROUTE2_MLNX_VERSION_FULL).dsc
+
+	if [ ! -z "$(MLNX_SDK_SOURCE_BASE_URL)" ]; then
+		# Build with SDK compiled from sources
+		python scripts/get_sdk_patches.py -i $(IPROUTE2_MLNX_VERSION) -u $(MLNX_SDK_SOURCE_BASE_URL)
+	else
+		# Build with SDK debians taken from github
+		python scripts/get_sdk_patches.py -i $(IPROUTE2_MLNX_VERSION) -s $(MLNX_SDK_VERSION)
```

Comment:
Please download in make file and pass the argument to script. Refer to PR below
```
https://github.com/vivekrnv/sonic-buildimage/pull/44
```

Reply:
I also think that's a better approach

Reply:
@vivekrnv @dgsudharsan 
I have removed the script completely, all operations performed done from Makefile.